guides/doc-Content_Management_Guide/topics/appendix-NFS_Share.adoc

```diff
@@ -27,35 +25,34 @@ Ensure this share provides the appropriate permissions to {ProjectServer} and it
 ----
 # {package-install-project} nfs-utils
 ----
-. You need to copy the existing contents of `/var/lib/pulp/content` to the NFS share.
+. You need to copy the existing contents of `/var/lib/pulp` to the NFS share.
 First, mount the NFS share to a temporary location:
 +
 ----
 # mkdir /mnt/temp
-# mount -o rw nfs.example.com:/satellite/content /mnt/temp
+# mount -o rw nfs.example.com:/satellite/pulp /mnt/temp
 ----
 +
-Copy the existing contents of `/var/lib/pulp/content` to the temporary location:
+Copy the existing contents of `/var/lib/pulp` to the temporary location:
 +
 ----
-# cp -r /var/lib/pulp/content/* /mnt/temp/.
+# cp -r /var/lib/pulp/* /mnt/temp/.
 ----
-. Set the permissions for all files on the share to use the `apache` user.
-This ID of this user is usually 48.
+. Set the permissions for all files on the share to use the `pulp` user.
 . Unmount the temporary storage location:
 +
 ----
 # umount /mnt/temp
 ----
-. Remove the existing contents of `/var/lib/pulp/content`:
+. Remove the existing contents of `/var/lib/pulp`:
 +
 ----
-# rm -rf /var/lib/pulp/content/*
+# rm -rf /var/lib/pulp/*
 ----
 . Edit the `/etc/fstab` file and add the following line:
 +
 ----
-nfs.example.com:/satellite/content    /var/lib/pulp/content   nfs    rw,hard,intr,context="system_u:object_r:httpd_sys_rw_content_t:s0"
+nfs.example.com:/satellite/pulp    /var/lib/pulp   nfs    rw,hard,intr,context="system_u:object_r:var_lib_t:s0"
```

Comment:
This part can be a little tricky with the SELinux labels. The issue here is that almost all of /var/lib/pulp uses `pulpcore_var_lib_t` except `/var/lib/pulp/pulpcore_static(/.*)?` which uses httpd_sys_content_t`. Static media doesn't get generated at runtime, so probably the best is to use `pulpcore_var_lib_t` and then have them relabel just that portion with an SELinux command after static media has been generated.

This is probably an area the selinux policy could be improved, but this is the current state of things.

Reply:
Oh an pulp2 does also use an area of /var/lib/pulp/ (not sure where off the top of my head) and I know that uses a different selinux context also. So that will need to be right as well.

Reply:
Oh yeah here's a link to the policy https://github.com/pulp/pulpcore-selinux/blob/master/pulpcore.fc#L17-L23

Reply:
This needs to be `system_u:object_r:pulpcore_var_lib_t`